client/src/components/TagSelector/NewTagButton.tsx

```diff
@@ -1,26 +1,26 @@
 import { useModalState } from "@/lib/state/modal-state";
-import { MdLabelImportant } from "react-icons/md";
+import { AiFillTags } from "react-icons/ai";
 import Modal from "../Modal";
 import NewTag from "../NewTag/NewTag";
 import * as S from "./NewTagButton.style";
 
 type NewTagButtonProps = {
 	modalId: string;
+	size?: number;
 };
 
-export default function NewTagButton({ modalId }: NewTagButtonProps) {
+export default function NewTagButton({ modalId, size = 15 }: NewTagButtonProps) {
```

Comment:
The default here is 15, but in the style file I have it at 30. What's the point?

Reply:
^ the 15 and 30 refer to different elements.